parser/include/parser/sourcemanager.h

```diff
@@ -32,16 +32,12 @@ class SourceManager
     bool operator()(model::FilePtr) const { return true; }
   };
 
-  template<typename Filter = SourceManager::AllFilesFilter>
-  std::vector<model::FilePtr> getFiles(const Filter& beta_ = Filter())
+  /**
+   * This function is a getter for the size of _files.
+   */
+  std::map<std::string, model::FilePtr>::size_type filesSize()
```

Comment:
I think `numberOfFiles()` or something similar would be better. This would also result a better documentation to the purpose of the function. And we should also modify the documentation above, because the user of this function doesn't know what `_files` is.

Reply:
Fixed in 80d9d2f2c410aad4feabf80cf21f1403c77f669f